### ISSUE TITLE
Bioproject id not found through API with Efetch ?? 

### ISSUE THREAD

**Issue by davidvilanova** [biopython/biopython#1125]
Hi,
I cannot find the following ids (although they are present at NCBI using the webinterface (database bioproject). The script takes a list of Boiprojects ids and returns the accesion numbers. Later in another script i get each dna sequence for each accession. 

In this case i do not have an associated accession so i cannot get the genome sequence.

What is the problem ??

The following bioproject id can be found through the webinterace (and has an accession associated)
[Id 211200](https://www.ncbi.nlm.nih.gov/bioproject/?term=211200)

```
import sys
from Bio import Entrez
from Bio import SeqIO

#define email for entrez login
db           = "nuccore"
Entrez.email = "someone@email.com"

#load accessions from arguments
if len(sys.argv[1:]) > 1:
  accs = sys.argv[1:]
else: #load accesions from stdin  
  accs = [ l.strip() for l in sys.stdin if l.strip() ]
#fetch
f = open('acclist', 'w')

sys.stderr.write( "Fetching %s entries for Bioprojects : %s\n" % (len(accs), ", ".join(accs[:10])))
for i,acc in enumerate(accs):
  try:
    sys.stderr.write( " %9i %s          \r" % (i+1,acc))  
    termtosearch = acc+"[BioProject]"
    handle = Entrez.esearch(db=db, term=termtosearch)
    record = Entrez.read(handle)
    for p in record: print record[p]

    ids = record['IdList']
    f.write('\n'.join(ids)+'\n') 
    f.flush()
    handle.close()

```

The ids cannot be found as seen in output
./bioproject_to_gi.py 246801 211200
```
Fetching 2 entries for Bioprojects : 246801, 211200
         1 246801          
0
0
[]
{u'OutputMessage': ['No items found.'], u'PhraseIgnored': [], u'QuotedPhraseNotFound': []}
{u'FieldNotFound': [], u'PhraseNotFound': ['246801[BioProject]']}
[]
0
(246801[BioProject])
0        2 211200          
0
[]
{u'OutputMessage': ['No items found.'], u'PhraseIgnored': [], u'QuotedPhraseNotFound': []}
{u'FieldNotFound': [], u'PhraseNotFound': ['211200[BioProject]']}
[]
0
(211200[BioProject])
Accession number saved to file acclist


```

**Comment by peterjc** [comment]
Note to self: These are not Entrez URLs, but they both work:

https://www.ncbi.nlm.nih.gov/bioproject/?term=246801
https://www.ncbi.nlm.nih.gov/bioproject/?term=211200

I see you closed this, did you solve this David?

**Comment by davidvilanova** [comment]
Hi Peter,
I realised that there is no assembly available for this bioproject yet at NCBI. I finally downloaded the assembly from the JGI Gold microbe database. which seems to be more up to date.

Thanks,

For instance the bioproject below will work as there is an assembly associated to it.
[https://www.ncbi.nlm.nih.gov/bioproject/?term=40683](https://www.ncbi.nlm.nih.gov/bioproject/?term=40683
)
